src/org/usfirst/frc/team4533/robot/utils/Arduino.java

```diff
@@ -2,10 +2,97 @@
 
 import org.usfirst.frc.team4533.robot.Robot;
 
+import edu.wpi.first.wpilibj.SerialPort;
+
 public class Arduino {
-	public double heading = 0;
-	
-	public static void update(){
-		Robot.heading = 0;
+	private static SerialPort serial;
+	private static boolean ready = false;
+	private static String oldBuffer = "";
+
+	private static final char syncChar = '^';
+	private static final char separatorChar = '~';
+
+	/* Initialize serial communications to the Ardunio */
+	public static void initialize() {
+		serial = new SerialPort(115200, SerialPort.Port.kUSB);
```

Comment:
Is there a constant that can be used for the baud rate?  If not, then no worries, but I thought there was a `SerialPort.BaudRate.115200` or something like that.